quickwit/quickwit-ui/src/components/IndexSideBar.tsx

```diff
@@ -37,22 +37,28 @@ function IndexAutocomplete(props: IndexMetadataProps) {
   const [open, setOpen] = React.useState(false);
   const [options, setOptions] = React.useState<readonly IndexMetadata[]>([]);
   const [value, setValue] = React.useState<IndexMetadata | null>(null);
-  const loading = open && options.length <= 1;
+  const [loading, setLoading] = React.useState(false);
+  // We want to show the circular progress only if we are loading some results and
+  // when there is no option avaiable.
+  const showLoading = loading && options.length == 0;
```

Comment:
Nitpick: Use `===` rather than `==` for the length comparision.

Reply:
oh thanks, you're right... I always forget that :/